src/main/java/io/github/eocqrs/cmig/State.java

```diff
@@ -22,19 +22,17 @@
 
 package io.github.eocqrs.cmig;
 
+import io.github.eocqrs.cmig.sha.StateChanges;
 import org.cactoos.Scalar;
-/*
- * @todo #32:30m/DEV design State interface
- */
 
 /**
  * State.
  *
  * @author Aliaksei Bialiauski (abialiauski.dev@gmail.com)
  * @since 0.0.0
  */
-public interface State extends Scalar<String> {
-
+public interface State extends Scalar<StateChanges> {
```

Comment:
@l3r8yJ can you provide simple implementation of this interface?

Reply:
@l3r8yJ it will help to resolve this one: https://github.com/eo-cqrs/cmig/issues/36

Reply:
@l3r8yJ also I have a suggestion for the design of this interface: we can add:
```java
Author author();
```

That will return a particular ChangeState's author from XML file

WDYT?

Reply:
@h1alexbel I think `Author author()` is an object with outside state, which I really don't like, wdyt?

I'm not sure about the implementation yet, I think I'll create a puzzle, wdyt?

Reply:
@l3r8yJ ChangeState has files a.k.a StateChanges and Author of it
ok, go ahead and create a puzzle